frontend/src/bundles/common/components/sub-navigation/sub-navigation.tsx

```diff
@@ -0,0 +1,48 @@
+import { PlusIcon } from '@heroicons/react/20/solid';
+
+import { ComponentSize } from '~/bundles/common/enums/component-size.enum.js';
+
+import { Button, ButtonVariant } from '../components.js';
+import { SubNavItem } from './components/sub-nav-item.js';
+
+type Properties = {
+    items: { label: string; to: string }[];
+    title?: string;
+    button?: { label: string; onClick: () => void };
+};
+
+const SubNavigation = ({ items, title, button }: Properties): JSX.Element => {
+    const bgColors = [
+        'bg-lm-yellow-100',
+        'bg-lm-magenta',
+        'bg-lm-purple',
+        'bg-lm-green',
+    ];
+
+    return (
+        <div className="bg-lm-black-200 flex h-full w-[20rem] flex-col gap-[1.75rem] overflow-auto p-[2.5rem]">
+            {title && <h1 className="text-xl font-bold text-white">{title}</h1>}
+            {items.map((item, index) => (
+                <SubNavItem
+                    key={index}
```

Comment:
I'm not sure if this is a good practice to add `key` as item index because if we add/delete workouts the usage of index will force the whole subnavigation to rerender and also there could be some unexpected behavior if the order of items changes and indexes change as well. What do you think?

Reply:
Hmm, you are right. So, you suggest adding id field for item type as we will have this ids with workouts?

Reply:
Yes  something like this